docs/testing.md

```diff
@@ -1,13 +1,14 @@
 # Bisq Testing Guide
 
+
```

Comment:
Could you please remove this empty line. Sometimes Travis gets stuck - in that case just leave a comment and I'll trigger a manual rebuild.

Reply:
Done. Please rebuild.

Reply:
continuous-integration/travis-ci/pr — The Travis CI build failed

Reply:
Just re-triggered a Travis build